Makefile

```diff
@@ -38,9 +38,9 @@ test: clean
 	-r requirements/app/test.txt
 
 	# run tests with coverage
-	python -m coverage run --source src/pytorch_lightning -m pytest src/pytorch_lightning tests/tests_pytorch -v
-	python -m coverage run --source src/lightning_app -m pytest tests/tests_app -v
-	python -m coverage run --source src/lightning_fabric -m pytest src/lightning_fabric tests/tests_fabric -v
+	python -m coverage run --source src/lightning/pytorch -m pytest src/lightning/pytorch tests/tests_pytorch -v
```

Comment:
we shall really have validation of makefile :(

Reply:
Yes #13468